### ISSUE TITLE
No rebooking capibility

### ISSUE THREAD

**Issue by jevers76** [dolson334/banterCustomer#8]
There should be a means to re-book a cabin and boat rental to the correct date slot in the following.  It is also nice to be able to show the proposed dates for next year at checkout time or on an invoice email

**Comment by dolson334** [comment]
@jevers76 we implemented the note search that we spoke of. This is a solid work around until the rebooking feature is fully functional.

![image](https://user-images.githubusercontent.com/16144247/153525050-54376f64-f6a2-4291-9fc2-b6bd75033319.png)

**Comment by jevers76** [comment]
The lack of re-booking can be solved for now using Notes as long as there is the capability to search the notes for some special text like 'REBOOK'.  When the guest is checking out and wants to rebook and there isn't time to make the rebook then 'REBOOK' can be added to the notes.  Later a search will be used to locate them.  I think I will possibly use the notes API to show the date for the following years time slot.

**Comment by dolson334** [comment]
@jevers76 right, and you have the ability now to do that with our search functionality.

**Comment by jevers76** [comment]
I am not going to close, but mark it deferred so that you will be reminded to add re-booking later
